unified-codes/apps/web/src/components/explorer/ExplorerTable.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+
+import { EntityTable } from '@unified-codes/ui';
+
+import { withStyles } from '../../styles';
+import { IState, ITheme, overflow } from '../../types';
+
+import ExplorerTableHeader from './ExplorerTableHeader';
+import ExplorerTableRows from './ExplorerTableRows';
+import ExplorerTablePagination from './ExplorerTablePagination';
+
+const styles = (theme: ITheme) => ({
+    tableContainer: {
+        marginTop: 5,
+        maxHeight: `calc(100vh - $370px)`,
```

Comment:
Is it used anywhere else, e.g. any particular reason why `370` should be `const` over e.g. `5` in `marginTop`? My thinking is that for styles would be good to move towards more consts in general, but I'm not sure if there's a particular motivation for this one?

Reply:
nah - not used elsewhere. it's just that the number is a fudge and is likely to change, so I thought that it would be good to lift it out of the code and put at the top of the method, y'know, where variable declarations belong.
Even better would be to remove the need for it.. I think we should work toward that instead, which probably means putting the pagination at the bottom of the table and not having it sticky on-screen.